transaction/core/src/ring_ct/rct_bulletproofs.rs

```diff
@@ -95,6 +99,39 @@ pub struct SignedInputRing {
     pub signed_digest: Option<[u8; 32]>,
 }
 
+/// Signature data that is required for any signer to produce signature
+/// and construct transaction
```

Comment:
Maybe this helps explain it better?

```suggestion
/// A structure that contains all the data required to sign a transaction that does not rely on having
/// the spend private key.
/// Exposing this allows implementing various offline/hardware signing solutions. The idea is that the data
/// in this object can be generated by only having the view private key for a set of input TxOuts, and it
/// can then be moved to a separate machine/service that then takes it + the spend private key and uses that
/// to generate a fully signed transaction.
```